mappings/net/minecraft/block/AbstractFireBlock.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/class_4770 net/minecraft/block/AbstractFireBlock
 	FIELD field_22088 damage F
 	FIELD field_22498 BASE_SHAPE Lnet/minecraft/class_265;
+	FIELD field_31008 ENTITY_ON_FIRE_FOR I
```

Comment:
Maybe `SET_ON_FIRE_TICKS`?

Reply:
The constant is in seconds, so I named it `SET_ON_FIRE_SECONDS`